### PR TITLE
Configure redux state

### PR DESCRIPTION
**Pull request by KozlovaAngelika**
Configure redux state

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +16,14 @@
         "@types/history": "4.7.3",
         "i18next": "20.3.5",
         "i18next-http-backend": "1.3.0",
+        "npm": "7.20.5",
```
Comment (brusnev-egor) [inline package.json:19]: ?

File: package.json [hunk 2]
```diff
@@ -32,7 +34,10 @@
         "@types/jest": "26.0.23",
         "@types/react-native": "0.64.5",
         "@types/react-native-i18n": "2.0.0",
+        "@types/react-redux": "^7.1.18",
```
Comment (brusnev-egor) [inline package.json:37]: remove ^

File: src/App.tsx [hunk 3]
```diff
@@ -5,16 +5,20 @@ import {
 } from 'react-native';
 import { useTranslation } from 'react-i18next';
 import './utils/i18n/index';
+import { Provider } from 'react-redux';
+import state from './redux/index';
```
Comment (brusnev-egor) [inline src/App.tsx:9]: `... from './redux' `

File: src/redux/redux.d.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+interface Movie {
+  id: number
+  title: string
+  poster: string
+}
+
```
Comment (i-nemets) [inline src/redux/redux.d.ts:6]: missing semicolon

File: src/redux/redux.d.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+interface MovieState {
+  movies: Movie[],
+  top250: Movie[],
+  favorites: Movie[]
+}
```
Comment (i-nemets) [inline src/redux/redux.d.ts:11]: missing semicolon

File: src/redux/redux.d.ts [hunk 6]
```diff
@@ -0,0 +1,11 @@
+interface Movie {
+  id: number
+  title: string
+  poster: string
+}
+
+interface MovieState {
+  movies: Movie[],
```
Comment (brusnev-egor) [inline src/redux/redux.d.ts:8]: please, add member-delimiter-style rule

Comment (brusnev-egor) [reply src/redux/redux.d.ts:8]: should be this rule
{
    "multiline": {
        "delimiter": "semi",
        "requireLast": true
    },
    "singleline": {
        "delimiter": "semi",
        "requireLast": false
    }
}

File: src/redux/redux.d.ts (hunk 4 above)
Comment (brusnev-egor) [reply src/redux/redux.d.ts:6]: it is not necessary to add semi after interface declaration. would be better to use semi as member delimiter for interfaces and types

**Review by brusnev-egor — APPROVED** [review]

**Review by i-nemets — APPROVED** [review]

**Pull request merged**